### DISCUSSION TITLE
How do I Uninstall?!?

### DISCUSSION THREAD

**Discussion by turbomaster95** [sdrausty/termux-archlinux#91] [Q&A] [answered]
I cannot uninstall it
Pls help
**And Pls Help Me SDRausty Pls Make A Uninstall Script 🥺**
Screenshot :
![Screenshot_2022-03-10-18-16-18-175](https://user-images.githubusercontent.com/86113389/157664948-8f38d86f-8cb7-4dd6-b289-5a3a6884d36e.jpeg)

**Comment by SDRausty** [comment, marked as answer]
The command `bash setupTermuxArch purge` uninstall the install directory.

```
PURGE    command 'setupTermuxArch purge' uninstalls Arch Linux in PRoot from Termux.

[purge [customdir]]  Purge/Uninstall Arch Linux from device.

[p[urge] [customdir]] Quick purge/uninstall Arch Linux from device.

SYNTAX[2]: [DO (help|install|manual|purge|refresh|sysinfo (default: install))]
```

The command `bash setupTermuxArch help` has more information.  Thank you for using `setupTermuxArch`, and why are you uninstalling Linux from your phone?
